### DISCUSSION TITLE
[Dashboard] Update Scores Chart to include number of scores on secondary y axis

### DISCUSSION THREAD

**Discussion by charlieviettq** [langfuse/langfuse#2282] [Ideas]
### Describe the feature or potential improvement

- Use a dual-axis bar chart to update the score chart, including the number of score records in the right column. 
   This enhancement will clarify the data and make it more useful for monitoring evaluation results.

**Comment by marcklingen** [comment]
Agree that the number of scores over time is interesting but I fear that the chart you mention will get very busy if we add this as some teams have lots of different scores plotted in this chart already. We'll be looking into additional score-related charts to add very soon. Please share any further ideas that you have
